keyboards/reviung/reviung41/keymaps/via/keymap.c

```diff
@@ -22,28 +22,32 @@ enum layer_names {
     _ADJUST
 };
 
+#define LOWER  MO(_LOWER)
+#define RAISE  MO(_RAISE)
+#define ADJUST MO(_ADJUST)
+
```

Comment:
```suggestion
```

Reply:
I modified the code and checked that it works with the keyboard, but the _ADJUST layer doesn't work.
I tried adding "TRI_LAYER_ENABLE = yes" to the rules.mk file, but it still doesn't work.
Where should I check?

Reply:
via enables tri layer support anyways.  

Also, if you have the layer_state_set_user function still, that will override the keycodes, and prevent them from working.

Reply:
Sorry, It seems that this fix was a misunderstand on my part.
I will close this pull request.
Thanks.